examples/h2o/h2o.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=h2o
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/h2o --conf=/etc/h2o/h2o.conf --m worker
```

Thank you for the changes. Does it make sense to:
* be consistent to using short options, and
* start in master mode so that the server can be reconfigured gracefully.
```suggestion
ExecStart=/usr/local/bin/h2o -c /etc/h2o/h2o.conf -m master
```

Otherwise the PR looks good to me.